src/udev/net/link-config.h

```diff
@@ -84,6 +85,8 @@ struct LinkConfig {
         int autoneg_flow_control;
         netdev_coalesce_param coalesce;
         uint8_t mdi;
+        CPUSet rps_cpu_mask;
+        bool set_rps_cpu_mask;
```

Comment:
How about to make it a pointer: `CPUSet rps_cpu_mask` -> `CPUSet *rps_cpu_mask`? Then,
- Default or setting an empty string: free the `CPUSet` object if exists,
- "no": allocate a CPUSet object if not, and unset all CPU bits,
- "all": allocate a CPUSet object if not, and set all existing CPU bits,
- list: allocate a CPUSet if not, and set flags for specified CPUs.

And then, set the attribute with the generated mask string only when `rps_cpu_mask` is non-NULL.
In this way, people can disable RPS even if enabled. Also, the boolean flag can be dropped.

Reply:
Sorry, "pointer noob" here. It failed in `fuzz-link-parser` after I changed from `CPUSet rps_cpu_mask` -> `CPUSet *rps_cpu_mask`.

Reply:
Some help would be greatly appreciated.

Reply:
I've changed it to pointer, @yuwata ptal.

I came from GC-featured language, I'm not confident if the pointer will be freed correctly.